### ISSUE TITLE
Question about "Self-supervised"

### ISSUE THREAD

**Issue by ZHANGH83** [junshengzhou/3D-OAE#2]
Dear zhou, congratulations to your great work!
I have a little question about the self-supervised part. Since the input pcd is already complete and the randomly masked parts are used for the loss calculation, maybe the prediction of the masked part is actually a supervised process? The masked parts actually have the corresponding ground truths, isn't it?
I'm not sure if I understand your work correctly, and really looking forward to your reply!

**Comment by junshengzhou** [comment]
Hi,

Thanks for your interest! The self-supervised learning is to directly leverage the point cloud for pretraining without requiring ground truth labels (e.g. category label, segmentation label). We of course need some supervision for training, but the supervision of recovering the masked parts comes from the point cloud itself, which makes it self-supervised.
